.ci/Jenkinsfile

```
@@ -383,7 +383,8 @@ def generateFunctionalTestStep(Map args = [:]){
                       powershell label: "Check for running elasticsearch", script: "Test-NetConnection ${linuxIp} -Port 9200"
                       powershell label: "Check for running kibana", script: "Test-NetConnection ${linuxIp} -Port 5601"
                       powershell label: "Check for running fleet server", script: "Test-NetConnection ${linuxIp} -Port 8220"
-                      cmd script: "cd e2e\\_suites\\${suite} && go test -timeout 60m -v --godog.tags=\"${tags}\"", label: "Run functional tests for ${platformLabels}:${suite}:${tags}"
+                      cmd script: "mkdir ${BASE_DIR}/outputs"
+                      cmd script: "cd e2e\\_suites\\${suite} && go test -timeout 60m -v --godog.format=\"junit:${BASE_DIR}/outputs/TEST-windows-${suite}.xml\" --godog.tags=\"${tags}\"", label: "Run functional tests for ${platformLabels}:${suite}:${tags}"
```

NIT: you can use the dir step, though it expects a body since it's a closure

```suggestion
                      dir("${BASE_DIR}\\outputs") {
                        cmd script: "cd ..\e2e\\_suites\\${suite} && go test -timeout 60m -v --godog.format=\"junit:${BASE_DIR}/outputs/TEST-windows-${suite}.xml\" --godog.tags=\"${tags}\"", label: "Run functional tests for ${platformLabels}:${suite}:${tags}"
                        }
```

---

BTW, no strong opinion, so whatever you like most 💯